src/Microsoft.AspNet.Mvc.ModelBinding/Validation/DefaultModelValidatorProvider.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Mvc.ModelBinding.Validation
+{
+    public class DefaultModelValidatorProvider : IModelValidatorProvider
```

Comment:
We have 10-15 validator implementations, and 3 providers.

1). Default provider - this is for extensibility - if you add an attribute that **is-a** `IModelValidator` this handles it for you - this is consistent with some of our designs for other metadata+provider concepts (filters, action constraints).
2). DataAnnotations provider, handles `ValidationAttribute` and `IValidatableObject`
3). DataMember provider, handles `[DataMember(IsRequired = true)]`

Would like to replace 3 with something better, but for now this is the best way to adopt the current validation system to the new provider extensibility.